thermodynamics/thermo_potential.py

```diff
@@ -0,0 +1,16 @@
+from pyiron_workflow import Workflow
+
+@Workflow.wrap.as_function_node()
+def obtain_potential(folder:str,pot_str:str):
```

Comment:
Best practice: because we've decorated this, it's _actually_ a class definition! To help visually identify node definitions, I recommend to use the usual class naming scheme, i.e. `ObtainPotential`

Reply:
`folder:str` -- unused?